lib/Field/FieldInstructions.js

```diff
@@ -0,0 +1,59 @@
+import React, { useContext } from 'react';
+import { func, number, string } from 'prop-types';
+import Linkify from 'linkifyjs/react';
+import { FieldNew } from 'lib';
+import ExpandingTextArea from '../ExpandingTextArea';
+
+export const FieldInstructions = ({
+  children,
+  onChange,
+  minRows,
+  placeholder,
+  className
+}) => {
+  const { inStructureEditMode } = useContext(
+    FieldNew.InStructureEditModeContext
+  );
+
+  if (inStructureEditMode) {
+    return (
+      <ExpandingTextArea
+        type="text"
+        className={`field__instructions
+          text-sm
+          border-0 border-t rounded-none
+          ${className}`}
+        handleOnChange={onChange}
+        value={children}
+        placeholder={placeholder}
+        minRows={minRows}
+      />
+    );
+  }
+
+  if (children) {
+    return (
+      <div className={`field__instructions text-sm ${className}`}>
+        <Linkify>{children}</Linkify>
+      </div>
+    );
+  }
+
+  return null;
```

Comment:
💚 Can we get rid of the `if`, and just return at 35. I feel like this would be ok, as we have defaulted children to an empty string.

Reply:
Hmm I don't want to render the <div> (which has a border top, padding etc) if we don't have any instructions. I could change it so it's up to the client to not render a FieldInstructions, but I think that might look a bit messy in the client code, something like
```jsx
fields.map(field => {
  return <FieldNew>
    <FieldContent
      instructions={field.instructions
        ? (<FieldNew.Instructions>{field.instructions}</FieldNew.Instructions>)
        : null
      }
    />
  </FieldNew>
}
```

whereas currently we can just do 
```jsx
fields.map(field => {
  return <FieldNew>
    <FieldContent
      instructions={<FieldNew.Instructions>{field.instructions}</FieldNew.Instructions>}
    />
  </FieldNew>
}
```
even when there are no instructions (just "")